<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>400</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>666</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>203</x>
      <y>-10</y>
      <width>7</width>
      <height>10086</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>173</y>
      <width>70</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Wellcome</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>140</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Press Enter to Start</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="scoreLCD">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>100</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>160</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Level ï¼š</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>80</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Score :</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="levelLCD">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>180</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>300</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGame_G">
    <property name="title">
     <string>Game</string>
    </property>
    <addaction name="actionStart_S"/>
    <addaction name="actionPause_P"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuPreference_S">
    <property name="title">
     <string>Preference</string>
    </property>
    <widget class="QMenu" name="menuSpeed">
     <property name="title">
      <string>Speed</string>
     </property>
     <addaction name="action1"/>
     <addaction name="action2"/>
     <addaction name="action3"/>
     <addaction name="action4"/>
     <addaction name="action5"/>
    </widget>
    <widget class="QMenu" name="menuLevel">
     <property name="title">
      <string>Add Hard</string>
     </property>
     <addaction name="action1_2"/>
     <addaction name="action2_2"/>
     <addaction name="action3_2"/>
     <addaction name="action4_2"/>
     <addaction name="action5_2"/>
    </widget>
    <widget class="QMenu" name="menuColor">
     <property name="title">
      <string>Color</string>
     </property>
     <widget class="QMenu" name="menuColor_1">
      <property name="title">
       <string>Red</string>
      </property>
      <addaction name="actionRed"/>
      <addaction name="actionYellow_2"/>
      <addaction name="actionBlue_2"/>
      <addaction name="actionGreen"/>
     </widget>
     <widget class="QMenu" name="menuColor_2">
      <property name="title">
       <string>Blue</string>
      </property>
      <addaction name="actionRed_2"/>
      <addaction name="actionYellow_3"/>
      <addaction name="actionBlue_3"/>
      <addaction name="actionGreen_2"/>
     </widget>
     <addaction name="menuColor_1"/>
     <addaction name="menuColor_2"/>
    </widget>
    <addaction name="menuSpeed"/>
    <addaction name="separator"/>
    <addaction name="menuLevel"/>
    <addaction name="menuColor"/>
   </widget>
   <widget class="QMenu" name="menuScore">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHow_to_Play"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuGame_G"/>
   <addaction name="menuPreference_S"/>
   <addaction name="menuScore"/>
  </widget>
  <action name="actionStart_S">
   <property name="text">
    <string>Start</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionPause_P">
   <property name="text">
    <string>Pause</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </action>
  <action name="action1">
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="action2">
   <property name="text">
    <string>2</string>
   </property>
  </action>
  <action name="action3">
   <property name="text">
    <string>3</string>
   </property>
  </action>
  <action name="action4">
   <property name="text">
    <string>4</string>
   </property>
  </action>
  <action name="action5">
   <property name="text">
    <string>5</string>
   </property>
  </action>
  <action name="action1_2">
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="action2_2">
   <property name="text">
    <string>2</string>
   </property>
  </action>
  <action name="action3_2">
   <property name="text">
    <string>3</string>
   </property>
  </action>
  <action name="action4_2">
   <property name="text">
    <string>4</string>
   </property>
  </action>
  <action name="action5_2">
   <property name="text">
    <string>5</string>
   </property>
  </action>
  <action name="actionRow">
   <property name="text">
    <string>Row</string>
   </property>
  </action>
  <action name="actionColumn">
   <property name="text">
    <string>Column</string>
   </property>
  </action>
  <action name="actionColor">
   <property name="text">
    <string>Color</string>
   </property>
  </action>
  <action name="actionRow_2">
   <property name="text">
    <string>Row</string>
   </property>
  </action>
  <action name="actionRow_Column">
   <property name="text">
    <string>Set Row &amp;&amp; Column</string>
   </property>
  </action>
  <action name="actionRed">
   <property name="text">
    <string>Red</string>
   </property>
  </action>
  <action name="actionYellow_2">
   <property name="text">
    <string>Yellow</string>
   </property>
  </action>
  <action name="actionBlue_2">
   <property name="text">
    <string>Blue</string>
   </property>
  </action>
  <action name="actionGreen">
   <property name="text">
    <string>Green</string>
   </property>
  </action>
  <action name="actionRed_2">
   <property name="text">
    <string>Red</string>
   </property>
  </action>
  <action name="actionYellow_3">
   <property name="text">
    <string>Yellow</string>
   </property>
  </action>
  <action name="actionBlue_3">
   <property name="text">
    <string>Blue</string>
   </property>
  </action>
  <action name="actionGreen_2">
   <property name="text">
    <string>Green</string>
   </property>
  </action>
  <action name="actionHow_to_Play">
   <property name="text">
    <string>How to Play</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
